### PR TITLE
Project Survey - Petra Brinck 

### PR DESCRIPTION
**Pull request by petrabrinck**
Link to my deployed project: https://modest-hopper-c5f89b.netlify.com

My reflections on how this week's project turned out: Pretty good I think. We started up this week together (me ad Theresa) which felt great. We got stuck when the submit button wouldn't work and that was because we divided the components (like we did with the music project) but Jennie solved it with us. 

Things I'd like to have clarified or explained in more detail:
I still battle with different turn outs depending on the browser I look in when I deploy so I am confused :-)

### CODE REVIEW

File: code/src/App.js [hunk 1]
```diff
@@ -1,9 +1,114 @@
-import React from 'react'
+import React, { useState } from 'react'
+import { Summary } from './Summary'
+
+const adventureGroup = ["surfing", "climbing", "hiking"];
```
Comment (ericaglimsholt) [inline code/src/App.js]:
```suggestion
const adventureGroups = ["surfing", "climbing", "hiking"];
```

File: code/src/App.js [hunk 2]
```diff
@@ -1,9 +1,114 @@
...
+const destinationGroup = ["Costa Rica", "Mount Everest", "Machu Picchu"];
```
Comment (ericaglimsholt) [inline code/src/App.js]:
```suggestion
const destinationGroups = ["Costa Rica", "Mount Everest", "Machu Picchu"];
```

File: code/src/App.js [hunk 3]
```diff
@@ -1,9 +1,114 @@
...
   return (
     <div>
-      Find me in src/app.js!
+
+      {!submitted && (
+
+        <form onSubmit={event => event.preventDefault()}>
+
+          <div className="heroHeader">
+            <h2>Plan your 2020 adventure!</h2>
```
Comment (ericaglimsholt) [inline code/src/App.js]:
```suggestion
            <h1>Plan your 2020 adventure!</h1>
```

File: code/src/App.js (hunk 1 above)
Comment (petrabrinck) [reply code/src/App.js]: Great, thanks! :-)

File: code/src/App.js (hunk 2 above)
Comment (petrabrinck) [reply code/src/App.js]: Great again, thanks! :-)

File: code/src/App.js (hunk 3 above)
Comment (petrabrinck) [reply code/src/App.js]: I started out with H1 but it looked huge on my laptop, but maybe its the best choice when it comes to accessibility?

**Comment by petrabrinck** [comment]
Answers: 
Re: Roboto font as files instead of including it as a link from Google fonts? 
I didn't know where to put it in React and I forgot to ask/double check with someone.   


Also - what do you mean by this?
"I still battle with different turn outs depending on the browser I look in when I deploy so I am confused"

When I look at this project on my laptop, on my Iphone XS, in Safari and in Chrome, the content is showing with different margins so I don't now if I should rely on one (or none) of them? :-/ 
Thank you! 
Petra

**Pull request closed without merging**